smart-contracts/contracts/BridgeBank/BridgeBank.sol

```diff
@@ -10,6 +10,10 @@ import "./BankStorage.sol";
 import "./Pausable.sol";
 import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 
+import "hardhat/console.sol";
```

Comment:
Remove this it should not be in our production smart contracts.

Reply:
Done